### DISCUSSION TITLE
HACS Problema download Add-on integrazioni

### DISCUSSION THREAD

**Discussion by Andiyp** [indomus/forum#2777] [GENERALE - Discussioni generiche]
Buongiorno, premesso che sono alle prime armi con HomeAssistant, ho installato e configurato senza aver ricevuto nessun avviso di errore HACS. Quando però vado a richiamare dallo store una integrazione di qualsiasi genere ad esempio "Tapo Controller" e vado a cliccare sul pulsante "scarica" non mi parte nessun download e non ricevo nessun tipo di errore.. Come posso risolvere? Grazie!

**Comment by snakuzzo** [comment]
Se non fornisci ultteriori dettagli (tipo i log), impossibile aiutarti.

**Reply by Andiyp** [reply]
Ok Grazie ..comunque ieri sera ho risolto il problema che era legato al browser di chrome che non riconosceva come attendibile hacs e ne impediva il download delle integrazioni

**Comment by Ugrec82** [comment]
Buongiorno a tutti sto cercando di installare dei componenti su hacs ma ricevo sempre questo errore. Team Tracker Card
Version v0.14.3 will be downloaded

Una volta scaricato, si troverà in '/config/www/community/ha-teamtracker-card'
Need a different version?
Downloading vasqued2/ha-teamtracker-card with version v0.14.3 failed with (Could not download, see log for details)
In questo caso sto provando a scaricare la card di teamtracker.

**Reply by snakuzzo** [reply]
C'è scritto cosa devi fare...

```
see log for details
```

**Reply by Ugrec82** [reply]
Qualsiasi versione metto,non me la fa scaricare,come mai? Dove sbaglio?

**Reply by snakuzzo** [reply]
scusa...ma di cosa stai parlando ?

**Reply by snakuzzo** [reply]
T'ho detto che c'è scritto `see log for details` e tu vai a cercare altre versioni.
Devi guardare i log !!!!
